### PR TITLE
Bug: iOS - handle asset not found

### PR DESCRIPTION
**Pull request by dw2kim**
Handle when asset is not found + Throw `platformException` for the details

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -131,7 +130,16 @@ class PlayerWidget extends StatelessWidget {
               return Text("Loading player");
             case ConnectionState.done:
               if (snapshot.hasError) {
-                return Text("Error loading player");
+                return Column(
+                  children: [
+                    Text("Error loading player"),
+                    RaisedButton(
+                        child: Text("Go Back"),
+                        onPressed: () async {
```
Comment (erickzanardo) [inline example/lib/main.dart]: No need for this to be async

**Review by erickzanardo — COMMENTED** [review]
Thanks for this, there is only one detail to be fixed, I will test this and merge after the nit is fixed!

**Comment by dw2kim** [comment]
@erickzanardo Any updates on this one?

**Comment by erickzanardo** [comment]
> @erickzanardo Any updates on this one?

Sorry for the delay, have a lot on my plate atm, Hopefully I will be able to merge this this week!

**Review by erickzanardo — APPROVED** [review]

**Comment by erickzanardo** [comment]
Released on `0.0.4`, thanks again for the contribution!

**Pull request merged**